README.md

```diff
@@ -0,0 +1,112 @@
+# Recipe-App
+
+> The Budget tracker app is a full stack application. It's a functional website that shows the list of categories and transactions and empower readers to interact with them by adding new categories and transactions. Users can access the services of the application by creating a new account or by login in if they already have account.
+
+## Built With
+
+- Major languages (Ruby)
+- Framworks (Ruby on Rails)
+- Testing libraries(Rspec)
+- Markup (HTML)
+- Styles (CSS, Tailwind)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- A text editor(preferably Visual Studio Code)
+
+### Install
+- Ruby
+- Ruby on Rails
+- PostgresSQL
+- Rspec
+
+### Using it Locally
+
+- Clone the project
+
+```
+git@github.com:roshan-bajgain/budget-app.git
+cd budget-app
+```
+
+
+### Setup
+
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+Start server with:
+
+```
+rails server
+```
+
+Visit http://localhost:3000/ in your browser.
+
+### Run tests
+
+Install npm with:
+
+```
+npm i
+```
+
+Install rspec with:
+
+```
+bundle install
+```
+
+and
+
+```
+rails generate rspec:install
+```
+
+run the test with:
+
+```
+rspec spec
+```
+
+## Authors
+
+👤 **Roshan Bajgain**
+
+- GitHub: [@roshan-bajgain](https://github.com/roshan-bajgain)
+- Twitter: [@RoshanBajgain10](https://twitter.com/RoshanBajgain10)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/roshan-bazgain/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/roshan-bajgain/budget-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Original design: [Gregoire Vella](https://www.behance.net/gregoirevella) on [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)
+- Inspiration: Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] [REQIRED] A very good project 👍🏽. However, @roshan-bajgain you need to ensure that this mobile web application is deployed using Heroku, Netlify, or GitHub Pages in order to make this project more accessible by any user and also make the project more professional.

Reply:
@Whoistolu, Thank you for your suggestion